src/bridge/bridge-unix-sockets.cpp

```diff
@@ -73,98 +74,157 @@ BasicLocalClient client{};
 inline constexpr int BUFFER_SIZE = 1024;
 using ByteBuffer = std::array<uint8_t, BUFFER_SIZE>;
 ByteBuffer byteBuffer;
+}
+
+SlimeVRDriver::Bridge::Bridge(VRDriver *i_pDriver)
+        : m_pDriver(i_pDriver), m_pPipe(nullptr), m_pBuffer(nullptr),
+          m_eState(EState::BRIDGE_DISCONNECTED), m_bStop(false) {
+}
+
+SlimeVRDriver::Bridge::~Bridge() {
+    this->stop();
+}
+
+void SlimeVRDriver::Bridge::run() {
```

Comment:
This appears to consist entirely of platform independent code, but is duplicated between windows pipes and unix sockets.